docs/usage_guide.rst

```diff
@@ -414,7 +418,20 @@ Additionally below the overview you get a table containing all of the taxable ev
    :align: center
 
 
-Finally you can get a nice CSV export by pressing the "Export CSV" button. This export is meant to be imported into google sheets. Press the button and then choose a directory to write the CSV files to. Once done you can import the CSV files into google sheets via its import menu.
+Finally you can get a CSV export by pressing the "Export CSV" button. This export is meant to be imported into Google Sheets. Press the button and then choose a directory to write the CSV files to. Once done you can import the CSV files into Google Sheets via its import menu. Following are definitions for the document's columns
+
+- ``type`` is a string describing the type of action a user engaged in.
+- ``paid_asset`` is a string identifying the asset an action was paid in, e.g. in "I bought 1 ETH for 100 EUR", `EUR` is the  `paid_asset`.
+- ``paid_in_asset`` is a float specifying the number of `paid_asset`s involved in an action.
+- ``taxable_amount``: is a float specifying the amount of `paid_asset` needed to be taken into consideration when generating a tax report.
+- ``received_asset`` is a string identifying the asset of an action's yield, e.g. in "I bought 1 ETH for 100 EUR", `ETH` is the `received_asset`.
+- ``received_in_asset`` is a float specifying the number of `received_asset`s involved in an action.
+- ``net_profit_or_loss`` is a float specifying the amount of profit or loss an action accounts for given the selected accounting strategy (currently there's only FIFO). Note that it's value is based on a spreadsheet formula.
+- ``time`` is a string containing the date and time an action was executed.
+- ``is_virtual`` is a boolean describing if an action's `paid_asset` and `received_asset` are both crypto assets. As in many jurisdictions, crypto to crypto trades are taxable, the `is_virtual` column allows us to calculate the action's taxable amount given the crypto coin's rate against rotki's default currency, also known as `profit_currency (e.g. EUR).
```

Comment:
You can also link to the relevant section of the User Guide here with regards to setting the app's "default currency".